MAUI/Maps/tooltip.md

```diff
@@ -383,7 +487,7 @@ Data template can be used customize the tooltip view using [`ShapeTooltipTemplat
             };
             Binding binding = new Binding();
             binding.Source = grid.BindingContext;
-            binding.Path = nameof(Model1.Continent);
+            binding.Path = nameof(DataItem.Continent);
```

Comment:
The syntax is wrong. Please refer

https://github.com/essential-studio/maui-maps/blob/release/20.3.0.1/maui/src/maps/ShapeLayer/MapShapeLayer.cs#L211

Reply:
I have updated.